Running from arcus-10
2019-05-16 20:37:21.045943: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-05-16 20:37:23.803743: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: TITAN V major: 7 minor: 0 memoryClockRate(GHz): 1.455
pciBusID: 0000:02:00.0
totalMemory: 11.75GiB freeMemory: 11.34GiB
2019-05-16 20:37:23.803799: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-05-16 20:37:24.351135: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-16 20:37:24.351190: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-05-16 20:37:24.351199: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-05-16 20:37:24.351568: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10955 MB memory) -> physical GPU (device: 0, name: TITAN V, pci bus id: 0000:02:00.0, compute capability: 7.0)
{'alpha': 0.6292066661236396, 'dropout': 0.32481241781368453, 'lr': 0.00045486129980048205, 'leaky_relu': 0.3237705607518082, 'batch_norm': 1, 'num_layers': 7, 'layer_0': 151, 'layer_1': 203, 'layer_2': 273, 'layer_3': 293, 'layer_4': 305, 'layer_5': 327, 'layer_6': 37, 'layer_7': 441, 'max_concurrent': 3, 'P': 'arcus_gpu.p', 'q': 'arcus.q', 'l': "hostname='(arcus-1|arcus-2|arcus-3|arcus-4|arcus-5|arcus-6|arcus-7|arcus-8|arcus-9|arcus-10)'", 'env': '/home/jott1/Projects/SHERPA_EX/.profile', 'data': 'fluxbypass_aqua', 'net_type': 'normal', 'loss_type': 'weak_loss', 'batch_size': 2048, 'data_dir': '/baldig/chemistry/earth_system_science/', 'max_dense_layers': 8, 'epochs': 25, 'patience': 10}
Epoch 1/25
 - 2712s - loss: 796.5335 - mean_squared_error: 2314.2341 - val_loss: -3.3513e+03 - val_mean_squared_error: 3444.5493
Epoch 2/25
 - 675s - loss: -3.3840e+05 - mean_squared_error: 229636.3006 - val_loss: -1.3007e+06 - val_mean_squared_error: 865619.5254
Epoch 3/25
 - 749s - loss: -8.2097e+06 - mean_squared_error: 5467090.2609 - val_loss: -2.7451e+07 - val_mean_squared_error: 18198559.3378
Epoch 4/25
 - 703s - loss: -5.3830e+07 - mean_squared_error: 35804387.2558 - val_loss: -1.3155e+08 - val_mean_squared_error: 87296696.7325
Epoch 5/25
 - 674s - loss: -1.9960e+08 - mean_squared_error: 132772904.3248 - val_loss: -3.3012e+08 - val_mean_squared_error: 219174933.2609
Epoch 6/25
 - 696s - loss: -5.4453e+08 - mean_squared_error: 362265519.9049 - val_loss: -8.4424e+08 - val_mean_squared_error: 561074584.7941
Epoch 7/25
 - 685s - loss: -1.2288e+09 - mean_squared_error: 817583497.7881 - val_loss: -1.4680e+09 - val_mean_squared_error: 975520004.1021
Epoch 8/25
 - 675s - loss: -2.4387e+09 - mean_squared_error: 1622597110.6945 - val_loss: -2.9097e+09 - val_mean_squared_error: 1933761395.2049
Epoch 9/25
 - 689s - loss: -4.3977e+09 - mean_squared_error: 2925897863.0657 - val_loss: -5.5959e+09 - val_mean_squared_error: 3718907209.4184
Epoch 10/25
 - 685s - loss: -7.3903e+09 - mean_squared_error: 4916806817.4602 - val_loss: -9.3816e+09 - val_mean_squared_error: 6230679361.1384
Epoch 11/25
 - 2533s - loss: -1.1728e+10 - mean_squared_error: 7801959984.8291 - val_loss: -1.4137e+10 - val_mean_squared_error: 9386070252.8198
/pkg/python/3.6.1-centos7/lib/python3.6/site-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Using TensorFlow backend.
Done with trial 11
